src/translations.js

```diff
@@ -250,6 +261,7 @@ const wakatimeCardLocales = {
     cs: "Statistiky Wakatime",
     de: "Wakatime Status",
     en: "Wakatime Stats",
+    ml: "സമയ സ്ഥിതിവിവരക്കണക്കുകൾ",
```

Comment:
Similar to the comment above. Google translates states that this translates to `Time Stats`.